config/playground/helm/dev/helm-opensearch.yaml

```diff
@@ -692,6 +692,12 @@ securityConfig:
             - cluster:admin/opendistro/rollup/get
             - cluster:admin/opendistro/transform/explain
             - cluster:admin/opendistro/transform/get
+            # To enable read access to security analytics
+            - cluster:admin/opensearch/securityanalytics/rule/search
```

Comment:
As you are modifying all the configuration, can you add the following to cluster permission, so anonymous user can run sql join as well? Thanks 
            # For using sql join query
            - "indices:data/read/scroll*"

Reply:
yeah, added this permission for anonymous user in dev/prod/preview